packages/bloom/lib/commands/bloom/CARD.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { strict as assert } from 'assert';
+import testUtils, { GLOBAL } from '../../test-utils';
+import { transformArguments } from './CARD';
+
+describe('BF CARD', () => {
+    it('transformArguments', () => {
+        assert.deepEqual(
+            transformArguments('bloom'),
+            ['BF.CARD', 'bloom']
+        );
+    });
+
+    testUtils.testWithClient('client.bf.card', async client => {
+        assert.equal(await client.bf.add('bf1', 'item'), true);
```

Comment:
There is not need to validate the 1 case, we're testing the client, not the module..